invenio_app_rdm/fixtures/__init__.py

```diff
@@ -25,3 +26,16 @@ def run(self, force=False):
             [Path("./app_data/pages"), dir_ / "pages"],
             force,
         ).load()
+
+
+class FixturesEngine:
```

Comment:
minor: I have the feeling that we could merge the `Pages` under `FixturesEngine`, since there's no special config or difference between the two (unless there is a reason to run these two separately). Same would apply to the CLI commands... @jrcastro2 WDYT?